<launch>

  <node
    name="stabilizer_control_node"
    pkg="pandora_stabilizer"
    type="stabilizer_control_node" 
    respawn="false"
    output="screen"/>

</launch>
